website.js

```diff
@@ -395,6 +410,18 @@ app.get('/admin/modules', function(req, res) {
     });
 });
 
+app.get('/admin/poll', function(req, res) {
+    connection.query('select * from poll', function(err, result) {
+        res.render('admin/poll/index.html')
+    });
+});
+
+app.get('/admin/poll/create', function(req, res) {
```

Comment:
If you want to stick with your own convention, then stick to that. No need to change it if you don't want to, it's just smart to think about it 👍